ruby-challenges.rb

```diff
@@ -3,16 +3,41 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+# Pseudo---->
+# create a block named oddOrEven
+# pass in a num paramater
+# create an if conditional statement that uses the method .even? to determine if the number is even
+# if it is even output "#{num} is even"
+# create an else conditional statement if the number is odd
+# output "#{num} is odd"
+# invoke the function, pass in the variable and print the result to the console
+
 num1 = 7
 # Expected output: '7 is odd'
 num2 = 42
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
 
+def odd_or_even num 
+  if num.even? 
+    "#{num} is even"
+  elsif num.odd? 
+    "#{num} is odd"
+  end
+end
```

Comment:
Awesome!  Happy to see that you are using some of Ruby's built-in methods.  For a stretch, consider how your method will handle a different datatype for an argument like a string.